resources/js/components/home-footer.js

```diff
@@ -6,7 +6,6 @@ import '@spectrum-web-components/overlay/overlay-trigger.js'
 import { customElement } from 'lit-element'
 import './app-menu.js'
 import './app-menu-item.js'
-import { magic_url } from '../helpers.js'
```

Comment:
@Perumal605 , why did y. ou remove the magic_url helper? This was added on purpose to fix a bug where the magic URL may not be applied properly

Reply:
@incraigulous I have updated magic_url helper.